### PR TITLE
Syncing with develop and  VB Processor testing updates

### PR DESCRIPTION
**Pull request by A-KVerma**

### CODE REVIEW

File: src/views/mainView.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React from "react";
+import React, {useEffect} from "react";
```
Comment (triptu) [inline src/views/mainView.jsx]: why is this needed?

Comment (A-KVerma) [reply src/views/mainView.jsx]: I will remove it was doing some testing

**Review by triptu — APPROVED** [review]
